### PR TITLE
fix: E2E tests & visibility of "No spare parts found" indicator flaky + UI5 version update

### PR DESCRIPTION
**Pull request by sam-garland**
closes: https://jira.tools.sap/browse/CXSPA-2383
closes: https://jira.tools.sap/browse/CXSPA-2384

### CODE REVIEW

File: projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts [hunk 1]
```diff
@@ -25,14 +25,34 @@ export function configureDefaultProduct() {
   cy.get('cx-breadcrumb').should('contain', 'Craftex');
 
   cy.get('cx-epd-visualization-visual-picking-tab');
+
+  cy.get('button')
+    .contains('Allow All')
+    .click({
+      timeout: 10_000
```
Comment (Matejk00) [inline projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts]: @sam-garland just wondering if it's necessary to set up a timeout for each button click? If so, maybe we can assign a timeout value to some `const` and re-use that variable in all needed cases.

**Review by Matejk00 — CHANGES_REQUESTED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts (hunk 1 above)
Comment (sam-garland) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts]: The timeout is needed on at least some of the button clicks in order to get the tests to work.
I can reduce some of the redundancy in this code.

Comment (sam-garland) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts]: Actually I found that a fix for https://jira.tools.sap/browse/CXSPA-2384 resolves the underlying problem that the click timeouts were working around.

I have included a fix for https://jira.tools.sap/browse/CXSPA-2384 which eliminates some flakiness affecting one of the integration tests.

As a result of this change I have been able to remove the non-default click timeouts.

**Review by Matejk00 — DISMISSED** [review]

File: projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts (hunk 1 above)
Comment (Matejk00) [reply projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/epd-visualization/visual-picking-tab.ts]: Thank you @sam-garland, now looks good to me. I approved both PR's.

**Review by Matejk00 — APPROVED** [review]

**Pull request merged**